server/parsec/cli/testbed.py

```diff
@@ -212,19 +218,29 @@ async def test_drop(raw_organization_id: str, request: Request) -> Response:
 
 
 @asynccontextmanager
-async def testbed_backend_factory(server_addr: ParsecAddr) -> AsyncIterator[TestbedBackend]:
+async def testbed_backend_factory(
+    server_addr: ParsecAddr, with_postgresql: str | None
+) -> AsyncIterator[TestbedBackend]:
+    db_url = "MOCKED" if with_postgresql is None else with_postgresql
+    blockstore_config = (
+        MockedBlockStoreConfig() if with_postgresql is None else PostgreSQLBlockStoreConfig()
+    )
+    # Same as the defaults in `db_server_options`
+    db_min_connections = 1 if with_postgresql is None else 5
+    db_max_connections = 1 if with_postgresql is None else 7
```

Comment:
Would it be useful to log this config when running the testbed server?

Update: maybe actually all the BackendConfig should be logged at server startup (if it's not the case already)

Reply:
3141c68cb650bd27bac43690fbfc09a9cb84d1ee